README.md

```diff
@@ -45,6 +48,11 @@ to learn about basic usage.
 Learn more at our [documentation](https://detectron2.readthedocs.org).
 And see [projects/](projects/) for some projects that are built on top of detectron2.
 
+Alternatively, you can also choose to consume the models as a hosted API from these site(s):
```

Comment:
```
Live model demos:
```
is a better line IMO.

We should have it in only one place in a given README file.  Either near Ukraine badge or this section

Reply:
@Venkat2811 Softened it up a little.  Otherwise, we could just strike it -- of course, the badge would be great -- I'm just concerned we're not giving enough context within the documentation on what we're doing.

Reply:
Our PR will have lots of details about what Tiyaro does.

`Try on Tiyaro` phrase in our badge and sub-sections like `Live model demos:` or `DensePose pre-trained models demo:` are short and self explanatory and should be good enough I think.